recipe/meta.yaml

```diff
@@ -13,36 +13,37 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
   entry_points:
     - prospector = prospector.run:main
 
 requirements:
   build:
     - python
-    - setuptools
+    - pip
 
   run:
     - python
-    - pylint >=1.5.6
+    - pylint >=1.5.6,<2.0.0
     - pylint-celery >=0.3
     - pylint-django >=0.7.2
     - pylint-flask >=0.3
```

Comment:
Missed that these 3 ( `pylint-celery`, `pylint-django`, `pylint-flask` ) were dropped as dependencies. Addressing this in PR ( https://github.com/conda-forge/prospector-feedstock/pull/8 ).

ref: https://github.com/PyCQA/prospector/commit/f866efc762c323779d283457abb86271b514d957#diff-2eeaed663bd0d25b7e608891384b7298